testsuite/cluster-test/src/instance.rs

```diff
@@ -65,6 +70,10 @@ impl Instance {
     pub fn ip(&self) -> &String {
         &self.ip
     }
+
+    pub fn port(&self) -> u32 {
```

Comment:
We should probably specify that this is `ac_port`, because there are many other ports for instance - ssh port, debug port, etc